frontend/src/components/pages/Profile.js

```diff
@@ -54,25 +54,36 @@ function Profile() {
   };
 
   const getEducations = (educations) => {
-    return educations.map((education) => (
-      <div className="mentor-profile-education">
-        <b>{education["school"]}</b>
-        <br />
-        {education["education_level"] + ", " + education["major"]}
-        <br />
-        {education["year"]}
-      </div>
-    ));
+    if (educations == null) {
+      return <div></div>;
+    }
```

Comment:
i'm not 100% sure what's going on with this function since it needs to be updated for when educations is a list but you should be able to combine this and the bottom check into (cont)
```suggestion
    if (educations == null || educations[0] == null ) {
      return <div></div>;
    }
```

Reply:
fixed!